### PR TITLE
[UI]: Add Dashboard responsiveness

### PR DESCRIPTION
**Pull request by dottharun**
**Notes for Reviewers**

This PR adds responsiveness to Dashboard UI

### Progress

- [x] Overview Tab.
- [x] Kubernetes Tabs.
- [x] Adjust dashboard gutters/padding.

#### Side effects fix:
- [x] Tool bar overflow in designs page.
- [x] Fixed-size patterns loading screen for `designs` page.
- [x] ActionModals cutoff issue.
- [x] Flip Card width issue.
- [x] InfoModal Grid correction.
- [x] Configurator Toolbar aligment fix. 

### Note:
- Input on design and styling are welcome

image

image

image

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Review by leecalcote — COMMENTED** [review]
@dottharun this is looking good so far. 

Charts aside, how are we doing on the Kubernetes dashboards?

Is this still a draft or are you ready for review?

**Comment by leecalcote** [comment]
> @dottharun this is looking good so far.
> 
> Charts aside, how are we doing on the Kubernetes dashboards?
> 
> Is this still a draft or are you ready for review?

@dottharun, are we tossing in the towel here?

**Comment by dottharun** [comment]
> Charts aside, how are we doing on the Kubernetes dashboards?
> 
> Is this still a draft or are you ready for review?

@leecalcote
Yes, Now fixed the issue.


https://github.com/meshery/meshery/assets/80959679/6d00aeb0-f281-43a0-9b11-49a35c1a80f9

**Comment by leecalcote** [comment]
@sudhanshutech have you reviewed?

**Review by leecalcote — DISMISSED** [review]
LGTM

File: ui/themes/app.js [hunk 1]
```diff
@@ -614,10 +614,18 @@ export const styles = (theme) => ({
     flex: 1,
     display: 'flex',
     flexDirection: 'column',
+    [theme.breakpoints.up('sm')]: {
+      width: `60%`,
+      overflowX: 'clip',
```
Comment (leecalcote) [inline ui/themes/app.js:619]: @dottharun, confirm that no other page is negatively affected by these broad-sweeping changes, please.

Comment (sudhanshutech) [reply ui/themes/app.js:619]: i didn't understand @dottharun , there is issue when using this breakpoint on other page right?

**Comment by leecalcote** [comment]
Ready for another review.

File: ui/themes/app.js (hunk 1 above)
Comment (dottharun) [reply ui/themes/app.js]: Yes.
- Now resolved 2 side-effects caused on other pages.
- Nothing more I could find caused by this PR.

**Comment by dottharun** [comment]
> Ready for another review.

Latest Demo after the fixes in the designs page:

https://github.com/meshery/meshery/assets/80959679/d00cba85-205d-4293-a93d-f8247ff9b5c8

**Review by dragon-slayer875 — CHANGES_REQUESTED** [review]
@dottharun please update the PR with the changes regarding padding.
If you need help let me know so that we get this merged quickly. 😁

**Pull request closed without merging**